frontend/src/views/OfferView.vue

```diff
@@ -29,13 +33,28 @@ let offerInfo = ref<OfferCreate | Offer>({
   items: [],
 });
 
+let offerItemsArray = ref<any>([]);
+
+function addOfferItem(type: ArticleKind) {
+  offerItemsArray.value.push({ id: id++, type: type });
```

Comment:
The ID should be generated by the backend, no?

Reply:
See the answer for v-for above.